Including some AWS Console access logging information (#2731)
CenturyLinkCloud/PublicKB: Cloud Application Manager/Deploying Anywhere/using-your-aws-account.md

|Service               | Type of service included in this instance. It can be an operating system, an application, a script, etc. |
|Hostname              | Hostname of the instance. |
|Provider              | [Provider](../Core Concepts/providers.md) in which this instance is deployed. |

> |Provider Instance ID  | AWS instance ID or IDs. If the user has enough rights, it shows a link to the resource or resources in the AWS console. Clicking this link will generate some log entries and show them within both instance’s and provider's activity logs. |

Reviewer: ```suggestion
|Provider Instance ID  | AWS instance ID or IDs. If the user has enough rights, it shows a link to the resource or resources in the AWS console. Clicking this link will register and show the action in both instance’s and provider's activity logs. |
```
